Client/API/ApiRequestHandler.cs

```diff
@@ -9,7 +9,7 @@ namespace Client.API
 {
     public class ApiRequestHandler : IApiHandler
     {
-        private string apiUrl = "https://localhost:7262";
```

Comment:
I can see you have changed the localhost address. can you please explain why you did that?